src/components/ClaimENSPage/ClaimENSPage.tsx

```diff
@@ -99,22 +112,16 @@ export default class ClaimENSPage extends React.PureComponent<Props, State> {
                 </span>
               </Section>
               <Form onSubmit={this.handleClaim}>
-                <Section>
+                <Section className={name.length === MAX_NAME_SIZE ? 'red' : ''}>
                   <Field
                     label={t('claim_ens_page.name_label')}
                     value={name}
-                    message={
-                      isError
-                        ? t('claim_ens_page.error_message')
-                        : isAvailable
-                        ? t('claim_ens_page.name_message')
-                        : t('claim_ens_page.repeated_message')
-                    }
+                    message={message}
                     placeholder={t('claim_ens_page.name_placeholder')}
                     action={`${name.length}/${MAX_NAME_SIZE}`}
                     error={isError || (hasNameMinLength(name) && !isValid) || !isAvailable}
                     onChange={this.handleNameChange}
-                    onAction={undefined}
+                    onAction={() => undefined}
```

Comment:
Hmmm, I'm not sure about this. @cazala  @nicosantangelo. What is the purpose of `onAction` ? `active = false` ?

Reply:
Fixing this with a class method to not regenerate the lambda on every render.